src/store/user/selectors.js

```diff
@@ -0,0 +1,7 @@
+import { compose, prop } from 'lodash/fp';
+
+const root = (state) => state.user;
+export const getUsername = compose(prop('username'), root);
+export const getType = compose(prop('userType'), root);
+export const getId = compose(prop('userId'), root);
```

Comment:
Same here, I think it's better to use names like getUserId, getUserType

Reply:
fixed